posthog/demo/matrix/manager.py

```diff
@@ -192,10 +195,12 @@ def save_sim_person(team: Team, subject: SimPerson):
             )
 
     @staticmethod
-    def save_sim_group(team: Team, type_index: Literal[0, 1, 2, 3, 4], key: str, properties: Dict[str, Any]):
-        from ee.clickhouse.models.group import create_group
+    def save_sim_group(
+        team: Team, type_index: Literal[0, 1, 2, 3, 4], key: str, properties: Dict[str, Any], timestamp: dt.datetime
+    ):
+        from ee.clickhouse.models.group import create_group_ch
 
-        create_group(team.pk, type_index, key, properties, clickhouse_only=True)
+        create_group_ch(team.pk, type_index, key, properties, timestamp)
```

Comment:
Q: Is this correct? Groups (similar to persons) rely on postgres and clickhouse being in sync. Getting them out of sync here feels dangerous.

I might be missing something though.

Reply:
They are still kept in sync in demo data! Just using ClickHouse as the source of truth. That data is then copied over to Postgres with the `sync_postgres_with_clickhouse_data` method. Reason is, the new demo environment scheme uses a master project (ID 0) to pre-save data for quick cloning, and this is a super straightforward scheme:

1. Copy all `team_id = 0` simulated usage to `team_id = {target_id}` within ClickHouse (events, persons, person distinct IDs, groups)
2. Sync Postgres state 1:1 for `team_id = {target_id}` from the `team_id = {target_id}` data in ClickHouse (persons, person distinct IDs, groups, event definitions, property definitions)

Reply:
Fair. Can we add a super scary warning to `create_group_ch` docstring that this should not be used outside of demo data as it can cause data corruption?

Reply:
Sure, renamed this to `raw_create_group_ch` and added a docstring with a warning